pipeline/lookup.csv

```diff
@@ -29678,3 +29678,892 @@ brownfield-land,,local-authority-eng:BAE,BFR53,1729669
+brownfield-land,,local-authority-eng:BRD,KY/180,1729793
+brownfield-land,,local-authority-eng:BRD,KY/181,1729794
+brownfield-land,,local-authority-eng:BRD,KY/184,1729795
+brownfield-land,,local-authority-eng:BRD,KY/185,1729796
+brownfield-land,,local-authority-eng:BRD,KY/189,1729797
+brownfield-land,,local-authority-eng:BRD,KY/192,1729798
+brownfield-land,,local-authority-eng:BRD,KY/193,1729799
+brownfield-land,,local-authority-eng:BRD,NE/004,1729800
+brownfield-land,,local-authority-eng:BRD,NE/081,1729801
+brownfield-land,,local-authority-eng:BRD,NE/112,1729802
+brownfield-land,,local-authority-eng:BRD,NE/178,1729803
+brownfield-land,,local-authority-eng:BRD,NE/182,1729804
+brownfield-land,,local-authority-eng:BRD,NE/184,1729805
+brownfield-land,,local-authority-eng:BRD,NW/054,1729806
+brownfield-land,,local-authority-eng:BRD,NW/125,1729807
+brownfield-land,,local-authority-eng:BRD,NW/127,1729808
+brownfield-land,,local-authority-eng:BRD,NW/129,1729809
+brownfield-land,,local-authority-eng:BRD,NW/134,1729810
+brownfield-land,,local-authority-eng:BRD,NW/137,1729811
+brownfield-land,,local-authority-eng:BRD,NW/145,1729812
+brownfield-land,,local-authority-eng:BRD,SE/048,1729813
+brownfield-land,,local-authority-eng:BRD,SE/054,1729814
+brownfield-land,,local-authority-eng:BRD,SE/079,1729815
+brownfield-land,,local-authority-eng:BRD,SE/111,1729816
+brownfield-land,,local-authority-eng:BRD,SE/185,1729817
+brownfield-land,,local-authority-eng:BRD,SE/188,1729818
+brownfield-land,,local-authority-eng:BRD,SH/052,1729819
+brownfield-land,,local-authority-eng:BRD,SH/058,1729820
+brownfield-land,,local-authority-eng:BRD,SH/061,1729821
+brownfield-land,,local-authority-eng:BRD,SH/062,1729822
+brownfield-land,,local-authority-eng:BRD,SH/063,1729823
+brownfield-land,,local-authority-eng:BRD,SH/064,1729824
+brownfield-land,,local-authority-eng:BRD,SH/065,1729825
+brownfield-land,,local-authority-eng:BRD,SH/067A,1729826
+brownfield-land,,local-authority-eng:BRD,SH/067B,1729827
+brownfield-land,,local-authority-eng:BRD,SH/068,1729828
+brownfield-land,,local-authority-eng:BRD,SH/074,1729829
+brownfield-land,,local-authority-eng:BRD,SH/084,1729830
+brownfield-land,,local-authority-eng:BRD,SI/024,1729831
+brownfield-land,,local-authority-eng:BRD,SW/035,1729832
+brownfield-land,,local-authority-eng:BRD,SW/088,1729833
+brownfield-land,,local-authority-eng:BRD,SW/157,1729834
+brownfield-land,,local-authority-eng:BRD,SW/158,1729835
+brownfield-land,,local-authority-eng:BRD,SW/160,1729836
+brownfield-land,,local-authority-eng:BRD,SW/163,1729837
+brownfield-land,,local-authority-eng:BRD,SW/165,1729838
+brownfield-land,,local-authority-eng:BRD,SW/168,1729839
+brownfield-land,,local-authority-eng:BRD,SW/170,1729840
+brownfield-land,,local-authority-eng:BRD,TH/037,1729841
+brownfield-land,,local-authority-eng:BRD,WI/002,1729842
+brownfield-land,,local-authority-eng:BRC,Map Ref 12 BRA7 Eastern Gateway Development Area, Bracknell (Town Square) (Submission Local Plan),1729843

```

Comment:
This and any other lookup where a reference has a comma in it need quotation marks